src/models/Projects.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const projectSchema = new mongoose.Schema({
+  employees: [
+    {
+      id: mongoose.SchemaTypes.ObjectId,
+      role: { type: String, require: true, enum: ['DEV', 'QA', 'TL', 'PM'] },
+      rate: { type: Number, require: true },
+    }],
```

Comment:
let's break this into two lines, as follows,
```suggestion
    }
  ],
```